Makefile

```diff
@@ -8,7 +8,7 @@ CSV2CPE=csv2cpe
 NVDSYNC=nvdsync
 RPM2CPE=rpm2cpe
 
-VERSION=1.0
+VERSION=$(TRAVIS_TAG)
```

Comment:
IIRC, this is the same as git tag of the commit. Meaning this can be empty. I think we need `on: tags: true` in `deploy` section of `travis.yml`

Reply:
Oh, it's already there, my bad. LGTM then.